docs/spark/tutorials/get-started-windows-instructions.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Get Started with .NET for Apache Spark on Windows
+description: Discover how to run a .NET for Apache Spark app using .NET Core on Windows.
+ms.date: 12/30/2019
+ms.topic: tutorial
+ms.custom: mvc
+# Customer intent: As a developer, I want to write a simple custom application using .NET for Apache Spark on Windows.
+---
+
+# Tutorial: Get Started with .NET for Apache Spark on Windows
+
+This tutorial will show you how to run a .NET for Apache Spark app using .NET Core on Windows.
+
+> [!div class="checklist"]
+>
+> * Pre-requisites to run your .NET for Apache Spark application
+> * Write a simple .NET for Apache Spark application
+> * Build and run your .NET for Apache Spark application
+
+## Pre-requisites
+
+- Download and install the following: **[.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** | **[Visual Studio 2019](https://www.visualstudio.com/downloads/)** | **[Java 1.8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)** | **[Apache Spark 2.4.1](https://archive.apache.org/dist/spark/spark-2.4.1/spark-2.4.1-bin-hadoop2.7.tgz)**
+- Download and install **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release:
+  - Select a **[Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases)** release from .NET for Apache Spark GitHub Releases page and download into your local machine (e.g., `c:\bin\Microsoft.Spark.Worker\`).
+  - **IMPORTANT** Create a [new environment variable](https://www.java.com/en/download/help/path.xml) `DOTNET_WORKER_DIR` and set it to the directory where you downloaded and extracted the Microsoft.Spark.Worker (e.g., `c:\bin\Microsoft.Spark.Worker`).
+
+For detailed instructions, you can see [Building .NET for Apache Spark from Source on Windows](../building/windows-instructions.md).
+
+## Authoring a .NET for Apache Spark App
+
+- Open Visual Studio -> Create New Project -> Console App (.NET Core) -> Name: `HelloSpark`
```

Comment:
```suggestion
1. Open Visual Studio > **Create New Project** > **Console App (.NET Core)** > Name: `HelloSpark`.
```

Reply:
These instructions seem to take a shortcut. Advanced users will probably know what you mean but not someone that is getting started.

Here's an example of a Get started with VS:
https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio

Reply:
Thanks for your suggestion! I've added a link.